examples/examples_load_optimization/02_sweep_values.py

```diff
@@ -0,0 +1,119 @@
+"""Example: Variable Operating Cost
+
+Demonstrate behavior of the Variable Operating Cost (VOC)
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from floris import FlorisModel, TimeSeries
+from floris.optimization.load_optimization.load_optimization import (
+    compute_net_revenue,
+    compute_turbine_voc,
+)
+
+
+# Compare VOC and net revenue under different conditions
+
+
+# Parameters
+D = 126.0
+N = 100
+
+
+# Declare a floris model with default configuration
+fmodel = FlorisModel(configuration="defaults")
+
+# Sweep TI first==================================================================
+
+wind_directions = np.ones(N) * 270.0
+
+# Assume uniform load ambient turbulence intensities
+load_ambient_tis = np.linspace(0.05, 0.25, N)
+
+# Assume uniform values
+values = 2e-5*np.ones(N) # $20/MWh
+
+# Declare a time series
+time_series = TimeSeries(
+    wind_directions=wind_directions, wind_speeds=8.0, turbulence_intensities=0.06, values=values
+)
+
+# Set the turbine layout to be a simple two turbine layout using the
+# time series object
+fmodel.set(layout_x=[0, D * 7], layout_y=[0.0, 0.0], wind_data=time_series)
+fmodel.run()
+
+# Compute the load turbulence intensity
+voc = compute_turbine_voc(fmodel, 2e-5, load_ambient_tis)
+
+# Compute net revenue
+net_revenue = compute_net_revenue(fmodel, 2e-5, load_ambient_tis)
```

Comment:
Note that I changed the ratio between A and values to make net revenue more sensitive to load ambient TI to make the example a little more interesting.